package.json

```diff
@@ -68,8 +69,9 @@
   },
   "dependencies": {
     "@aws-sdk/client-dynamodb": "3.651.1",
-    "@govuk-one-login/di-ipv-cri-common-express": "10.6.0",
+    "@govuk-one-login/di-ipv-cri-common-express": "10.7.1",
     "@govuk-one-login/frontend-analytics": "3.0.0",
+    "@govuk-one-login/frontend-device-intelligence": "^1.1.0",
```

Comment:
Can we remove the caret we have pinned the other dependency versions for the frontend

Reply:
corrected this and the others that where unpinned